pygmt/base_plotting.py

```diff
@@ -121,28 +122,88 @@ def coast(self, **kwargs):
             **+l** for lakes or **+r** for river-lakes, and passing multiple
             strings in a list.
         resolution : str
-            Selects the resolution of the data set to use ((f)ull, (h)igh,
-            (i)ntermediate, (l)ow, and (c)rude).
+            **f**\ |**h**\ |**i**\ |**l**\ |**c**
```

Comment:
Vertical bar (`|`) has special meanings in ReST, so you have to escape it. 

This is what it looks like now (note that the vertical bars are clickable)
![image](https://user-images.githubusercontent.com/3974108/105613002-b85ff380-5d8d-11eb-99a1-8d4e3ffdf352.png)

The correct syntax is:
```suggestion
            **f**\|\ **h**\|\ **i**\|\ **l**\|\ **c**
```

You can experiment it in the online ReST previewer (https://livesphinx.herokuapp.com/).

Reply:
@willschlitzer There are still some to be fixed, for example, 
![image](https://user-images.githubusercontent.com/3974108/105634733-9ae26680-5e2d-11eb-88fa-ef817f82c589.png).

And also see @weiji14's comment https://github.com/GenericMappingTools/pygmt/issues/631#issuecomment-764968391